### PR TITLE
cpp/leap: Mentor notes for C++ track Leap exercise

### PR DESCRIPTION
**Pull request by Llcoolsouder**

### CODE REVIEW

File: tracks/cpp/exercises/leap/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+### Common suggestions
+- There are just two cases that return `true`:
+  - a year is a multiple of 4 *and not** 100
+  - a year is a multiple of 4, 100, and 400
+- Don't use `if`/`else` branching to return a `bool`. Just return a boolean expression.
+- Don't try to rely on order of operations too much. Just use parentheses to make everything obvious.
+
+
+### Talking points
```
Comment (Llcoolsouder) [inline tracks/cpp/exercises/leap/mentoring.md:36]: Need some suggestions for this.

File: tracks/cpp/exercises/leap/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  {
+    return (year % 4 == 0) ^
+           (year % 100 == 0) ^
+           (year % 400 == 0);
+  }
+}  // namespace leap
+```
+
+### Common suggestions
+- There are just two cases that return `true`:
```
Comment (SleeplessByte) [inline tracks/cpp/exercises/leap/mentoring.md:29]:
```suggestion

- There are just two cases that return `true`:
```

File: tracks/cpp/exercises/leap/mentoring.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  - a year is a multiple of 4 *and not** 100
+  - a year is a multiple of 4, 100, and 400
+- Don't use `if`/`else` branching to return a `bool`. Just return a boolean expression.
+- Don't try to rely on order of operations too much. Just use parentheses to make everything obvious.
+- Make sure you enabled **all** tests.
+
+### Talking points
+- Argument name vs no argument name in declaration
```
Comment (SleeplessByte) [inline tracks/cpp/exercises/leap/mentoring.md:37]:
```suggestion

- Argument name vs no argument name in declaration
```

**Review by SleeplessByte — APPROVED** [review]
Just two formatting nits. Looking good!

**Comment by Llcoolsouder** [comment]
@ErikSchierboom I was told to tag you here.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
Merged! Thanks

**Pull request merged**